### ISSUE TITLE
Layout iteration - Leading story promo 1008+px

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#2208]
**Is your feature request related to a problem? Please describe.**
 We should iterate the first Story Promo (Feature 2) in a section for 1008+px.

**Describe the solution you'd like**
Since the Story Promo component is getting huge and the positions of the elements of this promo are different, we should create a new component that matches with UX designs 1008+px. This should have a developer accessibility swarm.

LTR:
![feature2](https://user-images.githubusercontent.com/46446236/65419505-b3d0a700-ddf6-11e9-9c2f-981c7c706cbf.png)

RTL:
![feature2-rtl](https://user-images.githubusercontent.com/46446236/65419571-d82c8380-ddf6-11e9-9f5a-cfe586957a62.png)

**Describe alternatives you've considered**
Reuse the current Story Promo component.

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [X] This feature is expected to need manual testing.

**Additional context**

**Comment by sareh** [comment]
The RTL version should be a mirror of the LTR version, so on the right you should see the story headline first. Would be good to get updated designs.

**Comment by amywalkerdev** [comment]
Blocked until grid work is done by the Shared Components pod
